appengine/logs/src/main/java/com/example/appengine/logs/LogsServlet.java

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START logs_API_example]
+package com.example.appengine.logs;
+
+import com.google.appengine.api.log.AppLogLine;
+import com.google.appengine.api.log.LogQuery;
+import com.google.appengine.api.log.LogServiceFactory;
+import com.google.appengine.api.log.RequestLogs;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Calendar;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+// Get request logs along with their app log lines and display them 5 at
+// a time, using a Next link to cycle through to the next 5.
+public class LogsServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest req, HttpServletResponse resp)
+         throws IOException {
```

Comment:
It takes the line to 91 columns and looks much wider than the rest of the page.  Since I'm expecting the line width to change to 80 as soon as I move it, I'm going to leave it that way for now.